src/DdSdkReactNative.tsx

```diff
@@ -48,6 +51,32 @@ export class DdSdkReactNative {
 
     }
 
+    /**
+     * Sets the global context (set of attributes) attached with all future Logs, Spans and RUM events.
+     * @param attributes: The global context attributes.
+     * @returns a Promise.
+     */
+    // eslint-disable-next-line @typescript-eslint/ban-types
+    static setAttributes(attributes: object): Promise<void> {
+        return new Promise<void>((resolve => {
+            DdSdk.setAttributes(attributes)
+            resolve()
+        }));
+    }
+
+    /**
+     * Set the user information.
+     * @param user: The user object (use builtin attributes: 'id', 'email', 'name', and/or any custom attribute).
+     * @returns a Promise.
+     */
+    // eslint-disable-next-line @typescript-eslint/ban-types
+    static setUser(user: object): Promise<void> {
+        return new Promise<void>((resolve => {
+            DdSdk.setUser(user)
```

Comment:
`DdSdk.setUser(user)` already returns a `Promise`, I'm not sure you need to wrap it in another promise here.